package.json

```diff
@@ -20,6 +20,7 @@
     "i18n",
     "materials/custom-properties.css",
     "materials/custom-properties.json",
+    "materials/custom-properties.js",
```

Comment:
Not sure it's necessary since we are exporting it now also as a named export from `src/index`

Reply:
Yeah I guess it's not really needed but we can keep it given anybody wants to copy it into their projects I suppose.